applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt_scan_for_conn_internal.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _BT_MGMT_SCAN_FOR_CONN_H_
+#define _BT_MGMT_SCAN_FOR_CONN_H_
+
+#include <zephyr/bluetooth/bluetooth.h>
+
+/**
+ * @brief	Scan for a connection with a given device name
+ *
+ * @param	scan_param	The scan parameters to use (interval and window)
```

Comment:
```suggestion
 * @param	scan_param	The scan parameters to use
```

Hard to maintain when specific variables are mentioned in the doc